CONTRIBUTING.md

```diff
@@ -690,6 +690,9 @@ Ask them nicely whether they still intend to review your PR and find yourself an
 
 - Improve skimmability: use a simple descriptive PR title (details go in commit titles) outlining _what_ is done and _why_.
 - Improve discoverability: apply all relevant labels, tick all relevant PR body checkboxes.
+- Add labels:
+  - If your PR needs a review, add the `needs_reviewer` label
```

Comment:
You could link to those labels, so people can see all the items with these labels. One caveat is that renaming the label will break the link.

Reply:
I didn't link on purpose, because the link isn't very useful. You can't add it to a PR from that page, and it doesn't show any description/details about the label. And since like you said it could break if the name is changed, on balance it seems better to not include it. I'd much rather have people just type it in; and if they get renamed to `9.needs: reviewer` like misuzu mentioned, github will still show that option when you type in `needs`